types/rpz/action.pp

```diff
@@ -0,0 +1 @@
+type Unbound::Rpz::Action = Enum['nxdomain', 'nodata', 'passthru', 'drop', 'disabled', 'cname']
```

Comment:
for both types, can you please add a `# @summary describe the type` and add a unit test? https://github.com/voxpupuli/puppet-unbound/tree/master/spec/type_aliases already has some examples.

Reply:
i added types for the struct seemed a bit overkill for the enums but can if you insist